frontend/src/components/Table/HorizontalTable.tsx

```diff
@@ -4,17 +4,30 @@ import {
   TablePagination,
   tablePaginationClasses as classes,
 } from "@mui/base/TablePagination";
+import { Button } from "../Button";
+import { Minus, Plus } from "lucide-react";
+import { ExpandRow } from "./ExpandRow";
 
 export interface HorizontalTableProps {
   header: string[];
   content: (string | number)[][];
+  alterRowColor?: boolean;
+  expandElement?: (idx: number) => React.ReactNode;
+  optionElement?: (idx: number) => React.ReactNode;
 }
 
-export function HorizontalTable({ header, content }: HorizontalTableProps) {
+export function HorizontalTable({
+  header,
+  content,
+  alterRowColor,
+  expandElement,
+  optionElement,
+}: HorizontalTableProps) {
   const [page, setPage] = React.useState(0);
   const [rowsPerPage, setRowsPerPage] = React.useState(10);
   const [sortBy, setSortBy] = React.useState("");
-  const [sortOrder, setSortOrder] = React.useState("desc");
+  const [sortOrder, setSortOrder] = React.useState("asc");
```

Comment:
It is now "desc" in master. Please merge `master` to this branch to get the latest updates